src/Management/src/Endpoint/Mappings/MappingsEndpoint.cs

```diff
@@ -239,7 +239,7 @@ private IList<string> GetHttpMethods(ApiDescription desc)
             };
         }
 
-        return null;
+        return new List<string>();
```

Comment:
Would it make sense to use `Enuerable.Empty<string>()` for cases like this?

Reply:
The return type is `IList<string>` and casting to `(IList<string>) Enumerable.Empty<string>();` results in "`Unable to cast object of type 'System.Linq.EmptyPartition'1[System.String]'' to type 'System.Collections.Generic.IList'1[System.String]'.`"

` Enumerable.Empty<string>().ToList();` would work, but I don't think that's better than `new List<string>()`;